doc/classes/TouchScreenButton.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TouchScreenButton" inherits="Node2D" version="4.0">
 	<brief_description>
-		Button for touch screen devices.
+		Button for touch screen devices for gameplay use.
 	</brief_description>
 	<description>
-		Button for touch screen devices. You can set it to be visible on all screens, or only on touch devices.
+		Button for touch screen devices for gameplay use.
+		This node inherits from [Node2D] and not [Control] so you can not set anchors on it. For UI buttons on touch screen devices use a [Button] node, and make sure emulate mouse from touch is enabled in project settings.
```

Comment:
I think this could be written in a more accessible way. Suggestion:

> This node inherits from [Node2D]. Unlike with [Control] nodes, you cannot set anchors on it. If you want to create menus or user interfaces, you may want to use [Button] nodes instead. To make button nodes react to touch events, you can enable the Emulate Mouse option in the Project Settings.